sdcm/cluster_k8s/__init__.py

```diff
@@ -464,6 +468,19 @@ def wait_for_pods_readiness(self, count: Optional[int] = None):
         if result.stdout.count('condition met') != count:
             raise RuntimeError('Not all nodes reported')
 
+    def statefulsets(self, namespace=None):
+        if namespace is None:
+            namespace = self.namespace
+        return KubernetesOps.list_statefulsets(self.k8s_cluster, namespace=namespace)
```

Comment:
Don't think we need this method here. It changes semantics of `None` value for `namespace` parameter and has very similar name to `list_statefulsets()`. Please, remove it and use `KubernetesOps.list_statefulsets()` below.

Reply:
done